components/validator/validator.go

```diff
@@ -175,3 +189,64 @@ func (v *Validator) Stop() error {
 
 	return nil
 }
+
+func (v *Validator) waitSyncCompleted() {
+	v.l.Info("start waiting for kroma node to sync")
+
+	ticker := time.NewTicker(time.Minute)
+	defer ticker.Stop()
+
+	for ; ; <-ticker.C {
+		select {
+		case <-v.ctx.Done():
+			return
+		default:
+			currentBlockNumber, err := v.fetchCurrentBlockNumber()
+			if err != nil {
+				continue
+			}
```

Comment:
Let's log error here as well.

Reply:
As I mentioned in PR description, I intended not to log sync status error while syncing because node always does not respond while syncing.

Reply:
Then, we need to remove Line 210, right?

Reply:
No, after pass `fetchCurrentBlockNumber`, `fetchLatestBlockNumber` must succeed. So if an error occurred, we need to log it.

Reply:
Is the node very slow to respond during sync, but it does respond?

Reply:
I may got your point.
`fetchLatestBlockNumber` is from L1, so if an error occurs, we need to log it.
It's optional, but then we can add `log.Debug` when error occurs in `fetchCurrentBlockNumber`.

Reply:
I added logging it with debug level.